src/kem/kyber/pqclean_kyber1024_aarch64/NTT_params.h

```diff
@@ -1,3 +1,30 @@
+
+/*
+ * CC0 1.0 Universal or the following MIT License
+ *
+ * MIT License
```

Comment:
This (dual license) is good(ness) in light of #1437. But then the contents of the PQClean aarch upstream algorithm YML seem to need updating (to also list MIT), no?